test/unit/view.child-views.spec.js

```
@@ -522,7 +522,7 @@ describe('layoutView', function() {
         war: '.craft',
         is: {
           regionClass: this.CustomRegion,
-          selector: '#a-fun-game'
+          el: '#a-fun-game'
```

these tests failed silently before 😿